CMakeLists.txt

```diff
@@ -68,37 +66,38 @@ elseif(DOXYGEN_FOUND)
 endif(DOXYGEN_FOUND)
 
 if (BUILD_PYTHON)
+  # This is NOT the modern cmake-included FindPython!
   find_package (Python COMPONENTS Development)
+
   message("Compiling Python ${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR} support. If this is not the intended target use 'DPYTHON_EXECUTABLE' to control.")
   # bind against boost-python for compilation of the python API
   # depending on major release version of the python libraries
-  if (${PYTHON_VERSION_MAJOR} VERSION_LESS "3")
-    find_package (Boost REQUIRED COMPONENTS python numpy)
-    find_package (Casacore REQUIRED COMPONENTS python)
-    message(STATUS "Boost python library: ${Boost_PYTHON_LIBRARY}")
-    message(STATUS "Boost numpy library: ${Boost_NUMPY_LIBRARY}")
-    message(STATUS "Python library ${PYTHON_LIBRARIES}")
-    set(BOOST_PY_FOUND TRUE)
-  else (${PYTHON_VERSION_MAJOR} VERSION_LESS "3")
-    find_package (Boost REQUIRED COMPONENTS python${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR} numpy${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR})
-    find_package (Casacore REQUIRED COMPONENTS python3)
-    message(STATUS "Boost python library: ${Boost_PYTHON${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}_LIBRARY}")
-    message(STATUS "Boost numpy library: ${Boost_NUMPY${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}_LIBRARY}")
-    message(STATUS "Python library ${PYTHON_LIBRARIES}")
-    set(BOOST_PY_FOUND TRUE)
-  endif (${PYTHON_VERSION_MAJOR} VERSION_LESS "3")
-
-
-  if(BOOST_PY_FOUND)
-      add_library(_stationresponse MODULE pystationresponse.cc)
-      set_target_properties(_stationresponse PROPERTIES PREFIX "")
-      target_include_directories(_stationresponse PRIVATE ${PYTHON_INCLUDE_DIR} ${Boost_INCLUDE_DIRS})
-      if (${PYTHON_VERSION_MAJOR} VERSION_LESS "3")
-        target_link_libraries(_stationresponse stationresponse ${CASA_PYTHON_LIBRARY} ${Boost_LIBRARIES} ${PYTHON_LIBRARIES})
-      else (${PYTHON_VERSION_MAJOR} VERSION_LESS "3")
-        target_link_libraries(_stationresponse stationresponse ${CASA_PYTHON3_LIBRARY} ${Boost_LIBRARIES} ${PYTHON_LIBRARIES})
-      endif (${PYTHON_VERSION_MAJOR} VERSION_LESS "3")
-      install(TARGETS _stationresponse DESTINATION ${PYTHON_INSTALL_DIR}/lofar/stationresponse)
-      install(FILES __init__.py DESTINATION ${PYTHON_INSTALL_DIR}/lofar/stationresponse)
-  endif(BOOST_PY_FOUND)
-endif (BUILD_PYTHON)
+  if(${PYTHON_VERSION_MAJOR} VERSION_LESS "3")
+    set(casapy "")
+    set(oldboostpy "")
+  else()
+    set(casapy "3")
+    set(oldboostpy "3")
+  endif()
+  set(boostpy ${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR})
+  message("trying boostpy=${boostpy}")
```

Comment:
This seems like a debug statement

Reply:
Thanks for the review, and sorry for merging too quickly. I think this debug output is desirable, since boost-python is leading to problems in almost all installations.

Reply:
Sure, no objections, but the message is a bit cryptic for a user. Might elaborate a little bit in the message.